docs/Makefile

```diff
@@ -69,6 +69,11 @@ pre-publish:
 .PHONY: publish-check
 	python3 redirects.py publish-check
 
+.PHONY: check-links
+check-links:
+	sphinx-build -b linkcheck . check-links -a
+	echo "link check complete, please see link-check/output.txt"
```

Comment:
It feels like we should be doing something with this in CI.  Would that be feasible?

Reply:
Yeah, we can just do a build and then print the contents of `output.txt`. I think that might need to be a separate ticket since that would break our CI until all 50+ issues are resolved.

Reply:
I'd agree that adding a separate PR with stuff fixed might be a good thing.

And that CI should run the check. People start to skip checks as soon as something "they didn't do" fails, and then never start again unless CI forces them.